packages/aws-cdk/lib/cli/root-dir.ts

```diff
@@ -1,6 +1,6 @@
 import * as fs from 'fs';
 import * as path from 'path';
-import { ToolkitError } from '../../../@aws-cdk/toolkit-lib/lib/api';
+import { ToolkitError } from '../../../@aws-cdk/toolkit-lib';
```

Comment:
Hmm - not caused by this PR, but why isn't this just:

```ts
import { ToolkitError } from '@aws-cdk/toolkit-lib`
```

?

Where does the relative path come from?

Reply:
Relative path as in import via the file system. We need this for imports that are not publicly exported from `@aws-cdk/toolkit-lib`. Mostly lower level code in `cdk-toolkit.ts` that is still needed while the CLI is not using `toolkit-lib`. 

I'm leaning towards keeping it consistent, and that means the relative import for now. Eventually (far future) the CLI would also just use what is publicly available, but we are not there yet.